### ISSUE TITLE
Incompatibility with `lcobucci/jwt:^5.0`

### ISSUE THREAD

**Issue by l-alexandrov** [corbosman/laravel-passport-claims#22]
### Summary
Laravel Passport 11 requires `"lcobucci/jwt": "^4.3|^5.0"`, but if we haven't specified a version lower than 5.0, the `CheckForClaim` middleware fails.

### Details
The `\Lcobucci\JWT\Configuration::forUnsecuredSigner()` method is deprecated since version 4.3 of the [`lcobucci/jwt`](https://github.com/lcobucci/jwt/blob/4.3.x/docs/upgrading.md#removal-of-none-algorithm) package. This method is used in the [`CheckForClaim` middleware](https://github.com/corbosman/laravel-passport-claims/blob/5.0.0/src/Http/Middleware/CheckForClaim.php#L29) of this package.

### Potential fix options
* Migrate to symmetric signer with dummy configuration like:
```
Configuration::forSymmetricSigner(
    new \Lcobucci\JWT\Signer\Rsa\Sha256(),
    \Lcobucci\JWT\Signer\Key\InMemory::plainText('empty', 'empty')
)
```
* Lock to `"lcobucci/jwt": "^4.3` in `composer.json`,

**Comment by corbosman** [comment]
I think this happened in a minor release of passport 11. When 11 came out it was locked on 4.3 so I didn't bother updating my code. Thanks for reporting, I'll look at it. I'll probably use the first option as that's what the JWT docs suggest.

**Comment by l-alexandrov** [comment]
Yes, it seems like it is a minor release done just yesterday. Since the parser is only used from the configuration, it could also be rewritten like [it's done in passport](https://github.com/laravel/passport/pull/1649/files#diff-b9d9183efdbe0ebe4b06bc23eac77b61825ceba1144aadcd40a2071ac0eebd37R299).
I'm going to propose changes.

**Comment by corbosman** [comment]
Hi, I released a new version 5.1.0 which I think should fix the issue. I installed a fresh laravel, fresh passport, and latest claims package and I correctly got claims on the JWT.  Let me know.

**Comment by l-alexandrov** [comment]
Yes, it works like a charm. Thanks!
